openj9.test.jlm/src/test.jlm.ibm/net/openj9/test/extensions/OperatingSystemExtensionCommand.java

```diff
@@ -1,152 +1,152 @@
+                    		+ "accessing the Proxy connetion for the OperatingSystemMXBean");
+				}
+				
+            } else { // We must be going via the MBeanServerConnection 
+            	try {
+            		ObjectName srvOSBeanName = new ObjectName(ManagementFactory.OPERATING_SYSTEM_MXBEAN_NAME);
+            		long free_physical_memory_size =  (long)mbs.getAttribute(srvOSBeanName, "FreePhysicalMemorySize");
+                    long process_virtual_memory_size =  (long)mbs.getAttribute(srvOSBeanName, "ProcessVirtualMemorySize");
+                    long process_private_memory_size =  (long)mbs.getAttribute(srvOSBeanName, "ProcessPrivateMemorySize");
+                    long process_physical_memory_size =  (long)mbs.getAttribute(srvOSBeanName, "ProcessPhysicalMemorySize");
+                    long process_cpu_time =  (long)mbs.getAttribute(srvOSBeanName, "ProcessCpuTime");
+                    checkOsInfo(free_physical_memory_size, process_virtual_memory_size,
+                    		process_private_memory_size, process_physical_memory_size, process_cpu_time);
+                } catch (IOException ioe) {
+                    ioe.printStackTrace();
+                    Assert.fail("A communication problem occurred when accessing the MBeanServerConnection");
+                    
+                } catch (MalformedObjectNameException mone) {
+                     mone.printStackTrace();
+                    Assert.fail("Problem with creating the ObjectName for a MXBean");
+                    
+                } catch (MBeanException mbe) {
+                    mbe.printStackTrace();
+                    Assert.fail("Problem with the MBean access");
+                    
+                } catch (AttributeNotFoundException ae) {
+                     ae.printStackTrace();
+                    Assert.fail("Attribute does not exist on the MBean");
+                    
+                } catch (InstanceNotFoundException ie) {
+                    ie.printStackTrace();
+                    Assert.fail("MBean Instance not found");
+                    
+                } catch (ReflectionException re) {
+                    re.printStackTrace();
+                    Assert.fail("Problem with the reflection of the MBean");
+                }
+            }
+        }
+    }
+
+    private void checkOsInfo(long free_physical_memory_size, long process_virtual_memory_size,
+    		long process_private_memory_size, long process_physical_memory_size, long process_cpu_time) {
+    	out.println("  Free Physical Memory Size:   " + free_physical_memory_size);
+    	out.println("  Process Virtual Memory Size: " + process_virtual_memory_size);
+    	out.println("  Process Private Memory Size: " + process_private_memory_size);
+    	out.println("  Process Physical Memory Size:" + process_physical_memory_size);
+    	out.println("  Process CPU time:  			" + process_cpu_time);
+    	out.println("");
+    	out.println("");
+    }
+}
```

Comment:
A number of the test.jlm files had Windows CRLF rather than unix line feed so I fixed them - but that caused the PR checker to insist on updating the copyright year.